### PR TITLE
[ADSS-397] Mobile Sticky Ad Reporting

### PR DESCRIPTION
**Pull request by anastasiiasok**
https://forbesmedia.atlassian.net/browse/ADSS-397

During the QA review, it was identified that when clicking on the mobile-sticky ad in the inspector, the highlight was appearing in the space right below the header. This behavior occurs because the highlight is positioned based on the top of the page when scrolling back. However, this is not the correct behavior, as we need better functionality to highlight the mobile-sticky ad at the bottom of the page and ensure it highlights correctly while scrolling. The original implementation only considered positioning the highlights for elements with **position: absolute**. This caused issues with highlighting elements that have **position: fixed**, such as the mobile-sticky ad.

This PR addresses the issue by adjusting the highlight overlay to account for **both absolute and fixed positioning**, ensuring the highlight is correctly aligned with the target element regardless of its position property.

Before:
image-20240509-160247

After:
Screenshot 2024-05-20 at 10 32 50 AM

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -139,10 +130,14 @@ const slotBuilder = (slot) => {
     head.append(utils.dom('span', {}, {}, name));
     const list = ulBuilder(params);
 
-    if (document.getElementById(slotHelper.getDomId(slot)) != null) {
-        const slotEl = document.getElementById(slotHelper.getDomId(slot));
+    const slotEl = document.getElementById(slotHelper.getDomId(slot));
+    //Outputting 'slotEl' to aid in debugging for verifying correct selection
+    console.log('slotElements:', slotEl);
```
Comment (mabramishvili) [inline src/app.js]: console.log should probably not be in prod code

Comment (anastasiiasok) [reply src/app.js]: @mabramishvili i left it on purpose to debug in the future all elements on a page, but sure i can remove
Screenshot 2024-05-15 at 2 15 35 PM

File: src/app.js [hunk 2]
```diff
@@ -161,6 +155,31 @@ const slotBuilder = (slot) => {
...
+        highStyle.position = 'fixed';
+    } else {
+        highStyle.position = 'absolute';
+    }
+
+    utils.setStyle(high, highStyle);
+};
+
+const scrollTo = (element) => {
+    const parentWrapper = element.closest('.ribbon-mobile-ad-wrapper');
```
Comment (sarahcalande) [inline src/app.js]: I think you would need mobile conditional here, this probably would not work on desktop right?

Comment (sarahcalande) [reply src/app.js]: unless this parentWrapper exists only on mobile; I would rename the variable like "mobileParentWrapper"

Comment (anastasiiasok) [reply src/app.js:174]: @sarahcalande this is only for mobile .ribbon-mobile-ad-wrapper

File: src/app.js (hunk 1 above)
Comment (mabramishvili) [reply src/app.js]: you can use debugger if you need to debug on prod environment for some reason, console.log statements shouldn't be used

**Review by GevKerobyan-OP — APPROVED** [review]

File: src/app.js (hunk 1 above)
Comment (anastasiiasok) [reply src/app.js]: removed

**Review by sarahcalande — APPROVED** [review]

**Pull request merged**